globus_cli/commands/endpoint/activate.py

```diff
@@ -1,14 +1,6 @@
 import click
 import webbrowser
 
-# only have --delegate-proxy appear if cryptography is available
-try:
-    import cryptography
-    # slightly hacky way of preventing flake8 from complaining
-    cryptography_imported = bool(cryptography)
-except ImportError:
-    cryptography_imported = False
-
```

Comment:
`globus_cli/services/transfer.py` also uses `cryptography_imported` and this detection code.

Reply:
Good catch. Will rectify. (Of course, all of that code still runs fine, just isn't necessary.)